Build error : M_PI’ undeclared

rm -f src/*.o ascii-view.x
gcc -Wall -Wextra -Wpedantic -std=c99 -Iinclude -O3 -flto -march=native -c src/argparse.c -o src/argparse.o
gcc -Wall -Wextra -Wpedantic -std=c99 -Iinclude -O3 -flto -march=native -c src/image.c -o src/image.o
gcc -Wall -Wextra -Wpedantic -std=c99 -Iinclude -O3 -flto -march=native -c src/main.c -o src/main.o
gcc -Wall -Wextra -Wpedantic -std=c99 -Iinclude -O3 -flto -march=native -c src/print_image.c -o src/print_image.o
src/print_image.c: In function ‘print_image’:
src/print_image.c:154:57: error: ‘M_PI’ undeclared (first use in this function)
  154 |             double sobel_angle = atan2(sy, sx) * 180. / M_PI;
      |                                                         ^~~~
src/print_image.c:154:57: note: each undeclared identifier is reported only once for each function it appears in
make: *** [makefile:18: src/print_image.o] Error 1

--- a/makefile
+++ b/makefile
@@ -1,6 +1,6 @@
 CC = gcc
-CFLAGS = -Wall -Wextra -Wpedantic -std=c99 -Iinclude
-LDFLAGS = 
+CFLAGS = -Wall -Wextra -Wpedantic -std=c99 -Iinclude -D_GNU_SOURCE
+LDFLAGS = -lm
 SRCDIR = src
 SOURCES = $(wildcard $(SRCDIR)/*.c)
 OBJECTS = $(SOURCES:.c=.o)

